virttest/virsh.py

```diff
@@ -1557,6 +1558,10 @@ def undefine(name, options=None, **dargs):
     if options is not None:
         cmd += " %s" % options
 
+    if platform.machine() == "aarch64":
```

Comment:
How about this solution: 
https://github.com/avocado-framework/avocado-vt/commit/3986c19fe69cec7fb06c8300ac2e42cfee0521c9

The configuration has already settled in machines.cfg.

Reply:
Hi 
> How about this solution:
> [3986c19](https://github.com/avocado-framework/avocado-vt/commit/3986c19fe69cec7fb06c8300ac2e42cfee0521c9)
> 
> The configuration has already settled in machines.cfg.
Emm, I am not sure whether we could get self.params here.

Reply:
Right. Then this is ok to me. Thanks.